packages/host/tests/integration/components/card-basics-test.gts

```diff
@@ -518,6 +523,61 @@ module('Integration | card-basics', function (hooks) {
       .containsText(`Module: http://localhost:4202/test/person Name: Person`);
   });
 
+  test('render base64 image card', async function (assert) {
+    let { field, contains, CardDef } = cardApi;
+    let { Base64ImageField } = base64Image;
+    class DriverCard extends CardDef {
+      @field image = contains(Base64ImageField);
+    }
+
+    let driver = new DriverCard();
+    await renderCard(loader, driver, 'edit');
+    triggerEvent('[data-test-base64-field]', 'change', {
+      files: [base64ToBlob(mango, 'image/png')],
+    });
+    await waitFor('[data-test-actual-img]');
+    await fillIn('[data-test-field="altText"] input', 'picture of mango');
+    assert
+      .dom('[data-test-actual-img]')
+      .hasAttribute('src', `data:image/png;base64,${mango}`);
+    assert
+      .dom('[data-test-actual-img]')
+      .hasAttribute('alt', 'picture of mango');
+
+    await click('[data-test-contain-size-input]');
+    assert
+      .dom('[data-test-height-warning]')
+      .includesText(
+        `Can't render current image. Please provide a height when using the "contain" or "cover" size.`,
```

Comment:
I think `.exists()` would be sufficient since the text isn’t dynamic, just the presence of its container.